common.py

```diff
@@ -0,0 +1,110 @@
+import csv
+import os
+import urllib.request
+
+from IPython import embed
+
+
+class FileLoaderMixin():
+
+    def load_file(self, file_path):
+
+        # make sure that we only try to load supported file types
+        supported_file_types = {'.csv', '.txt'}
+        current_file_type = file_path[file_path.rfind('.'):]
+        if not current_file_type in supported_file_types:
+            raise ValueError("The FileLoaderMixin currently supports {supported_file_types} but not "
```

Comment:
this should be an f-string.  (unfortunately, pylint doesn't seem to detect this yet; eslint does for the back-tick f-string equivalent in Javascript)

Reply:
fixed